sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatThreadAsyncClient.java

```diff
@@ -336,7 +336,7 @@ public String getChatThreadId() {
      * @return the {@link CompletableFuture} that emits list of thread participants in the first page.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public CompletableFuture<Page<ChatParticipant>> getParticipantsFirstPage() {
+    public CompletableFuture<Page<String, ChatParticipant>> getParticipantsFirstPage() {
```

Comment:
Note: Eventually, the public API in SDKs (e.g. Chat) will not return `Page<String, ..>` or `PagedResponse<..>`, see the `listChat()` as an example. The APIs that expose those types in this pr is the one needs to be adapted to new asyncstream / pagedIterable \\cc @CHILIU-MSFT